test/systemtests/kubeadm_test.go

```diff
@@ -0,0 +1,981 @@
+package systemtests
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/contiv/contivmodel/client"
+	"os"
+	"time"
+)
+
+type kubePod struct {
+	node *node
+}
```

Comment:
Could you add a `func (k *KubePod) isMaster() bool` here and call that instead of repeating `name == "k8smaster"` many times in this file?